docker_mcp/models/params.py

```
@@ -0,0 +1,215 @@
+    )
+    schedule_id: Optional[str] = Field(
+        default=None,
+        description="Schedule identifier for management **(used by: schedule remove/enable/disable)**"
+    )
+
+
+class DockerContainerParams(BaseModel):
+    """Parameters for the docker_container consolidated tool."""
+    
+    action: str = Field(
+        ..., 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)